extensions/mssql/src/contracts.ts

```diff
@@ -1614,4 +1614,12 @@ export namespace RenameObjectRequest {
 	export const type = new RequestType<RenameObjectRequestParams, void, void, void>('objectManagement/rename');
 }
 
+export interface DropObjectRequestParams {
+	connectionUri: string;
+	objectUrn: string;
```

Comment:
Add the doc comments here too

Reply:
We will be consolidating these APIs, will add doc later.